src/panels/config/automation/trigger/types/ha-automation-trigger-webhook.ts

```diff
@@ -12,8 +17,15 @@ export class HaWebhookTrigger extends LitElement {
   @property() public trigger!: WebhookTrigger;
 
   public static get defaultConfig() {
+    // The webhook_id should be treated like a password. Generate a default
+    // value that would be hard for someone to guess. This generates a
+    // 144-bit random value. The output is a 24 character url-safe string.
+    const randomBytes = crypto.getRandomValues(new Uint8Array(18));
+    const base64Str = btoa(String.fromCharCode(...randomBytes));
+    const urlSafeId = base64Str.replace(/\+/g, "-").replace(/\//g, "_");
```

Comment:
Users should know that they can still add a normal word to the end to make it easy to distinguish. So wonder if we should do that here, like add the automation name to the end (although we don't have that context here)

Reply:
ie `asbjkvsjhkcsajkhaaafzxc-hello-world`

Reply:
I like this idea a lot! It also helps remove one of the reasons users might have for replacing the random id.

My JS/TS skills aren't strong. Please check that I've done this correctly in [1ba271a](https://github.com/home-assistant/frontend/pull/11568/commits/1ba271a115ba078f692869730acdea100ea7f91e).
![image](https://user-images.githubusercontent.com/289218/152861403-0d18064b-e71e-4b82-94ed-21090d45e5ec.png)

Reply:
Thanks for the sanity check. I've reversed the order in [d1bd24f](https://github.com/home-assistant/frontend/pull/11568/commits/d1bd24fca68a37b21638424786cf57b36e58d561).

![image](https://user-images.githubusercontent.com/289218/152878594-788f56ca-1dd6-40ba-bb5f-df433719d0fb.png)

Reply:
It does get truncated a bit on Android.
![image](https://user-images.githubusercontent.com/289218/152880351-a332894a-9f29-46e1-884c-3e3e7dfe83c1.png)